docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/eksctl.md

```diff
@@ -0,0 +1,288 @@
+
+For the simplicity of this guide we'll be using non best practices but refer to further documents to allow you to delve deeper into the topic.
+
+## Outcome
+
+Following this guide will result in
+
+- an AWS EKS Kubernetes Cluster with 4 nodes and the possibility to scale up further.
+- the [EBS CSI driver](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) is installed and configured.
+- a [managed PostgreSQL](https://aws.amazon.com/rds/postgresql/) instance is spawned that will be used by the Camunda applications.
+
+This basic cluster setup is required to continue with the Helm setup as described in our [AWS Helm Guide](#). <!-- TODO: reference future guide -->
+
+## Usage
+
+The eksctl tool allows creating clusters via a single command but this doesn't support all configuration options. Therefore, we're supplying a `yaml` file that can be used with the CLI to create the cluster preconfigured with various settings.
+
+### Eksctl Prerequisites
+
+Setup authentication to allow interacting with `AWS` via the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) to configure access.
+
+General advice, the user creating the resources will be the owner of those and always linked to those. Meaning the user will always have admin access on Kubernetes, except you delete it. Therefore, it can make sense to create an extra [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) that's solely used for the EKSCTL purposes.
+
+<!--
+```shell
+eksctl create cluster --name test-cluster --region eu-central-1 --with-oidc --version 1.28 --nodegroup-name services --instance-types m5.xlarge --nodes 4 --nodes-min 1 --nodes-max 10 --node-private-networking --disable-pod-imds --vpc-cidr 10.192.0.0/16 --vpc-nat-mode HighlyAvailable --dry-run
+```
+-->
+
+### Eksctl Cluster yaml
+
+The following `yaml` has to be copied to a file on your system and can be executed with.
+
+```shell
+eksctl create cluster --config-file cluster.yaml
+```
+
+Where `cluster.yaml` contains the following contents.
+Before applying, please adjust the `<region>` and `<zone>` and rename the cluster name with the placeholder `<clusterName>` to your liking.
+
+```yaml
+apiVersion: eksctl.io/v1alpha5
+availabilityZones:
+  - <zone>c # e.g. eu-central-1c
+  - <zone>b
+  - <zone>a
+cloudWatch:
+  clusterLogging: {}
+iam:
+  vpcResourceControllerPolicy: true
+  withOIDC: true # enables and configures OIDC for IRSA
+addons:
+  - name: vpc-cni
+    resolveConflicts: overwrite
+    version: latest
+  - name: kube-proxy
+    resolveConflicts: overwrite
+    version: latest
+  - name: aws-ebs-csi-driver # automatically configures irsa
+    resolveConflicts: overwrite
+    version: latest
+  - name: coredns
+    resolveConflicts: overwrite
+    version: latest
+kind: ClusterConfig
+kubernetesNetworkConfig:
+  ipFamily: IPv4
+managedNodeGroups:
+  - amiFamily: AmazonLinux2
+    desiredCapacity: 4
```

Comment:
according to all the platform guides yes
https://docs.camunda.io/docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/

3 nodes alone are required because of 3 default Zeebe replicas and they have constraints to not spawn on the same node. I'd assume 1 additional node for misc.

With a bit of config adjustments one could probably run it with less as well.